kraken-std/src/kraken/std/python/pyproject.py

```diff
@@ -161,7 +161,7 @@ def get_name(self) -> str:
     def get_version(self) -> str | None:
         return cast(str, self._pyproj["project"].get("requires-python", None))
 
-    def upsert_source(self, source_name: str, url: str, _default: bool = False, _secondary: bool = False) -> None:
+    def upsert_source(self, source_name: str, url: str, priority: str = "") -> None:
```

Comment:
Can we use `IndexPriority` here as well?

Reply:
it creates a circular import